core/client/src/client.rs

```diff
@@ -1480,16 +1481,19 @@ impl<B, E, Block, RA> CallRuntimeAt<Block> for Client<B, E, Block, RA> where
 				self.execution_strategies.syncing.get_manager(),
 			ExecutionContext::Importing =>
 				self.execution_strategies.importing.get_manager(),
+			ExecutionContext::OffchainCall =>
+				self.execution_strategies.other.get_manager(),
+			ExecutionContext::RichOffchainCall(_) =>
+				self.execution_strategies.other.get_manager(),
```

Comment:
Should we modify the `CLI`?

Reply:
`Other` still encapsulates previous kinds of calls, I don't there is any reason to be so specific about when we call WASM and when not. Both `OffchainCall` and `RichOffchainCall` are only going to be used when calling into runtime from node and I think it's fine for them to have the same strategy.